Assets/!/Scripts/TwoDimensionalAnimationStateController.cs

```diff
@@ -13,12 +13,13 @@ public class TwoDimensionalAnimationStateController : MonoBehaviour
     public float maximumCrouchVelocity = 0.5f;
     public float maximumProneVelocity = 0.25f;
     public float maximumBackwardsVelocity = 0.5f;
+    public bool invincible = false;

```

Comment:
This should not be in an animation controller.
Please remove because there is a godMode param in #40 and we can enable that feature once both PRs are merged